recipe/meta.yaml

```
@@ -1,14 +1,14 @@
 {% set native = 'm2w64-' if win else '' %}
 {% set posix = 'm2-' if win else '' %}
-{% set version = "4.5.1" %}
+{% set version = "4.5.2" %}
 
 package:
   name: r-base-split
   version: {{ version }}
 
 source:
   - url: https://cran.r-project.org/src/base/R-4/R-{{ version }}.tar.gz
-    sha256: b42a7921400386645b10105b91c68728787db5c4c83c9f6c30acdce632e1bb70
+    sha256: 87a41ce9b50e096dd2c4282f48efea30c9916fcb7b167fa2bc988c9cf3cb6642
     patches:
       - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
```

```suggestion
      - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch  # [osx]
```

The linux and win builds are failing to apply this patch to the source code. Can we just apply it to the osx-* builds?

---

Patching seems resolved now.